frontend/src/styles/layout/_page.sass

```diff
@@ -16,127 +16,231 @@
   background: $color-engel-medium-light
   padding: $spacing-layout-s
   display: grid
-  grid-template-columns: 65% 35%
+  grid-template-columns: 70% 30%
+
   > .search
-    margin-bottom: $spacing-layout-s
+    margin-bottom: $spacing-s
 
 .results
   grid-row: 2
+
   &-list
     background: white
     list-style: none
     padding: 0
+    margin: 0
+
     > a
       display: block
+
       &:not(:last-child)
         border-bottom: 1px solid #ccc
+
     &__item
       cursor: pointer
       @include mix.flexbox
       justify-content: space-between
       padding: $spacing-layout-xs
-      &:hover
-        background-color: #ededed
+
+      &:not(:last-child)
+        border-bottom: 1px solid #ccc
+
+      .name, .address, .date, .number, .state, .details > div
+        @include mix.flexbox
+        flex-flow: column nowrap
+        justify-content: center
+
+
+      .name, .address, .date
+        width: 32%
+
+      .name, .address
+        margin-right: 2%
+
+      .date
+        text-align: right
+
+      .area
+        width: 15%
+
+      .state
+        width: 15%
+
       &--apartment
-        display: grid
+        display: flex
+        height: 100px
+        //padding-left: 0
         grid-template-columns: 90px auto
         grid-template-rows: auto auto
+
         .number
-          grid-column: 1
           font-size: $fontsize-heading-l
-          font-weight: 600
-          width: 90px
+          letter-spacing: -1px
+          text-transform: uppercase
+          width: calc(90px)
+          line-height: 1.25
+        //text-align: right
+        //align-items: center
```

Comment:
remove commented lines?

Reply:
Oops 😅 - leftovers from when I was trying to find a nice solution to styling it. Commented lines removed.